RecoEgamma/PhotonIdentification/plugins/PhotonIDValueMapProducer.cc

```diff
@@ -96,7 +96,7 @@ class PhotonIDValueMapProducer : public edm::global::EDProducer<> {
   // HiggsTo2photons anaysis code. Template is introduced to handle reco/pat
   // photons and aod/miniAOD PF candidates collections
   float computeWorstPFChargedIsolation(const reco::Photon& photon,
-                                       const edm::View<reco::Candidate>& pfCands,
+                                       const std::vector<edm::Ptr<reco::Candidate>> pfCands,
```

Comment:
```suggestion
                                       const std::vector<edm::Ptr<reco::Candidate>>& pfCands,
```
to avoid a vector copy.